src/demetriek/models.py

```diff
@@ -69,6 +69,7 @@ class Display(DataClassORJSONMixin):
 
     brightness: int
     brightness_mode: BrightnessMode
+    brightness_range: Range
```

Comment:
This doesn't exist on the older devices (Running firmware >2.1.0).


Also considering the documentation from the API and the PR description, aren't we looking for brightness_limit?

![CleanShot 2025-01-13 at 17 08 59](https://github.com/user-attachments/assets/c6d07d72-b3b2-4f88-a0f1-403147225571)

Reply:
Do you have one of the older ones and can you share fixtures?


![image](https://github.com/user-attachments/assets/937a6ac8-411c-44e9-aac4-52ed8a9497cf)

Oh wrong field indeed

Reply:
Unfortunately, the fixture in this repo are created using my oldest device